app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -20,7 +20,7 @@
       <% @presenter.options.each do |option| %>
         <%= label_tag "two_factor_options_form_selection_#{option.type}",
                       class: "btn-border col-12 mb2 #{option.html_class}",
```

Comment:
`btn-border` applies `display: inline-block`, which without the `!important` of BassCSS will override `display-none`'s `display: none`.

https://github.com/18F/identity-idp/blob/281a7b2098f5090d5cf8eda9d2df0b9e4347a829/app/assets/stylesheets/components/_btn.scss#L86

Thoughts which come to mind:

- Can we avoid `display: inline-block` in `btn-border`?
- Can we reorder loaded stylesheets so that `.display-none` of equal specificity to `.btn-border` "wins"?
- Should we have a custom variant of `display-none` which is "stronger", either via load order, CSS selector, or use of `!important` (ideally we avoid `!important`).

Reply:
In 31d29016a, I opted to remove the `display` style from `.btn-border`, verified by checking every instance of bordered buttons in the application that it wouldn't have any regressions. In cases where it would (only the account delete request screen), I added an explicit `display-inline-block`.